### ISSUE TITLE
Buy a domain name and point it at https://markafitzgerald1.github.io/cribbage-trainer/

### ISSUE THREAD

**Issue by markafitzgerald1** [markafitzgerald1/cribbage-trainer#163]
Hard to navigate to the Cribbage Trainer single page application at present due to all of low Google Search ranking (#22 to mitigate), link to SPA visually buried in `README.md` (#162 to mitigate), and hard to communicate address (https://markafitzgerald1.github.io/cribbage-trainer/).

**Comment by markafitzgerald1** [comment]
Claude Code (Fable 5) agent: Implementation note from the PR-preview work (#153/#661): a custom domain serves the site at the root path instead of /cribbage-trainer/, so this change must also update the Vite `base` default and the `PAGES_BASE_PATH` values in the deploy workflows, plus the preview URL construction (`.../pr/<number>/` links in the workflow and the `pages-content` state-branch layout assumptions). Also sequence this **before** #22 (Google indexing) so Google indexes the final domain rather than github.io URLs that would immediately need migrating.

**Comment by markafitzgerald1** [comment]
Codex (GPT-5.6 Sol, Extra High effort), on Mark's behalf:

Add one privacy-operational task when this domain is registered and adopted:

- Create a generic address such as `privacy@<domain>` and forward it to the current privacy inbox, `mark.a.fitzgerald+cribbage.trainer@gmail.com`.
- Replace the Gmail address in the in-app Privacy Policy and any Google Analytics data-processing/account contact fields with the domain address.
- Keep the old address working for a transition period so previously published policy copies remain actionable.
- Configure the domain mail records and forwarding so privacy requests are reliably delivered; include SPF, DKIM, and DMARC where supported.

This keeps the current PR unblocked while avoiding a long-term personal-address dependency.

**Comment by markafitzgerald1** [comment]
Claude Code (Sonnet 5, High effort): Moved from Minimum Lovable Product to Beyond MLP per a 2026-08-25 board-priority review with Mark. `AGENTS.md` states plainly that finding, retaining, and monetizing users is deliberately deferred until the roadmap's stickiness and trust gates hold, and that they don't yet -- this issue is a find-more-users action, so it belongs behind that gate rather than in the active milestone.
